### PR TITLE
jenkins: Add CentOS and Fedora jobs for kata 2.0

### LINKED ISSUE #332: Add jenkins jobs for centos and Fedora for kata 2.0

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the configs for the CentOS and Fedora jenkins jobs for
the kata 2.0 CI

Fixes #332

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by devimc — APPROVED** [review]
thanks @GabyCT

File: jenkins/jobs/kata-containers-2.0-centos-PR/config.xml [hunk 1]
```diff
@@ -0,0 +1,166 @@
+
```
Comment (jodh-intel) [inline jenkins/jobs/kata-containers-2.0-centos-PR/config.xml:1]: @GabyCT - Travis failed due to this blank line, so I think it's safest if you remove it:

```
INFO: Checking XML file 'jenkins/jobs/kata-containers-2.0-centos-PR/config.xml'
-:2: parser error : XML declaration allowed only at the start of the document
<?xml version='1.0' encoding='UTF-8'?>
     ^
ERROR: failed to check XML file 'jenkins/jobs/kata-containers-2.0-centos-PR/config.xml'
```

Comment (GabyCT) [reply jenkins/jobs/kata-containers-2.0-centos-PR/config.xml]: done :)

**Review by jodh-intel — APPROVED** [review]
Thanks @GabyCT.

lgtm

**Pull request merged**